### ISSUE TITLE
Extract multiple witnesses from Notes

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/MyopicVicar#812]
Some transcribers have entered 3rd and more witness in the notes field

**Comment by Sherlock21** [comment]
Yes, I have enter the overspill witnesses into the notes field on all occasions where there were more than 2, rather than create a second entry.

**Comment by Captainkirkdawson** [comment]
This requires a display and analysis of all notes fields in marriages

**Comment by edickens** [comment]
I think it will have to be done by hand, although a list of all entries with "Witness" or "Wit:" in the Notes could identify where to look.
There is also the situation where multiple entries have been made to accommodate the extra witnesses, which ideally should be combined into one entry.

**Comment by DeniseColbert** [comment]
Transcribers in the past (and also now in WinREG) have duplicated the entry for multiple witnesses and have placed extra witnesses in the Notes field. We need to show these and combine duplicates into one entry.

First: Rake task to read Notes field for Witness or Wit.

Then: decide how to deal with them.

Duplicate records: new story.
